configure

```diff
@@ -16398,6 +16398,14 @@ fi
 
 $as_echo "#define PYTHON /**/" >>confdefs.h
 
+  if test "x$with_python" == "x"; then :
+
+cat >>confdefs.h <<_ACEOF
+#define SYSTEM_PYTHON_PATH "$PYTHON"
+_ACEOF
+
+
+fi
```

Comment:
Why are we making this change? Is it related to this defect?

Reply:
BTW, is configure part of the checked-in source? I think not.

Reply:
In our 14.1 release branch, we used to keep GNU auto-generated files.

Reply:
We should not retain auto-generated files in any branch.

Reply:
Thanks for your suggestion @mike0042. 
Please guide me how can I adhere this, 
Shall I *ignore* auto-generated files in this PR ?  (or) Shall i raise another PR for *removal* of all other auto generated files?

Reply:
@bremanandjk it is fine to leave it the way this branch was before. Thanks for explaining.

Reply:
@bremanandjk there are two things we should be doing...
1. Keeping the list of files in .gitignore up to date, and
2. Remove any auto-generated files we find checked in to the repo.

Reply:
Thanks, @mike0042  I have implemented your comments.